sqli_helpers.go

```diff
@@ -65,9 +67,9 @@ func strLenSpn(s string, length int, accept string) int {
 	return length
 }
 
-func strLenCSpn(s string, length int, accept string) int {
+func strLenCSpn(s string, length int, accept []byte) int {
```

Comment:
@jptosso it would be really cool to know what this function does or why is it needed. As of now the name tells me nothing :(

Reply:
From what I recall, there is a general task of rewriting a lot of this repo into idiomatic Go, currently I guess it's mostly line-by-line point of C. Admittedly while touching this I probably should have renamed it 😅